scene/2d/camera_2d.cpp

```diff
@@ -415,6 +426,14 @@ void Camera2D::set_current(bool p_current) {
 	}
 }
 
+void Camera2D::_update_process_internal_for_smoothing() {
+	const bool is_not_in_scene_or_editor = !(is_inside_tree() && Engine::get_singleton()->is_editor_hint());
+	const bool is_any_smoothing_valid = follow_smoothing > 0 || rotation_smoothing_speed > 0;
+
+	const bool enabled = is_any_smoothing_valid && is_not_in_scene_or_editor;
```

Comment:
We don't use `const` for primitive types, because it's pointless (especially in such short methods).

Reply:
Going to keep it in mind. This code was left untouched during the rebasing.